backend/app/api/endpoints/statistics.py

```diff
@@ -11,8 +11,18 @@
 async def get_group_statistics(
     request: Request
 ):
+    access_token = request.raise_if_missing_authorization()
+
     app = request.app
+    user_id, _ = await app.ow_sync.sync_for_access_token(access_token)
+
     async with app.db.pool.acquire() as conn:
+        is_in_any_ow_group = app.db.groups.is_in_any_ow_group(user_id, conn=conn)
```

Comment:
her er det en liten bug: app.db.groups.is_in_any_ow_group er en coroutine som ikke blir await'et, så `if not is_in_any_ow_group` er alltid truthy (siden coroutinen ikke er None).